docs/integrations/microsoft-azure/opentelemetry/active-directory-json-opentelemetry.md

```diff
@@ -0,0 +1,140 @@
+---
+id: active-directory-json-opentelemetry
+title: Active Directory JSON - OpenTelemetry Collector
+sidebar_label: Active Directory JSON - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Active Directory JSON.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/windows.png')} alt="Thumbnail icon" width="40"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Active Directory](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/get-started/virtual-dc/active-directory-domain-services-overview) JSON App helps you monitor your Windows Active Directory deployment by analyzing Active Directory logs in the JSON based event log format. The app includes predefined searches and dashboards that provide user activity into your environment for real-time analysis of overall usage.
+
+We recommend using the Active Directory JSON App in combination with the Windows JSON App.Active Directory logs are sent to Sumo Logic through OpenTelemetry [windowseventlogreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/windowseventlogreceiver).
+
+<img src={useBaseUrl('https://https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Active-Directory-OpenTelemetry/Active-Directory-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Active Directory
```

Comment:
1. Use relative paths - `/docs/manage/fields/`
2. I see only a single field under this heading.